data/tools/kics.yml

```diff
@@ -0,0 +1,17 @@
+name: kics
+categories:
+  - linter
+tags:
+  - configmanagement
+license: Apache License 2.0
+types:
```

Comment:
Except for `cli`, these list entries are actually `tags`. Guess CI should fail in this case. Our generator is not enforcing the types to exist to make it possible to add a new type in the future without having to touch the code, but maybe we should change that. 
Anyway, will move it over to the tags section.